recipes/serpent/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "1.12" %}
+
+package:
+  name: serpent
```

Comment:
Technically this is `Serpent`. We haven't really decided what we do with casing in names though.

@ocefpaf care to weigh in on this one too?